index.js

```diff
@@ -82,7 +82,7 @@ server.delete("/api/users/:id", (req, res) => {
       );
       // res.json({ message: `User with ID: ${id} successfully deleted.` });
     })
-    .catch(() => {
+    .catch(err => {
```

Comment:
you wrote the err into this catch but it's just a param for the caught error, so to get the benefits of it don't forget to return it.